lib/ex_doc/formatter/html.ex

```diff
@@ -162,7 +162,9 @@ defmodule ExDoc.Formatter.HTML do
 
   defp default_assets(config) do
     debug = if config.debug, do: [{Assets.debug(), "dist"}], else: []
-    [{Assets.dist(), "dist"}, {Assets.fonts(), "fonts"} | debug]
+    [{Assets.dist(), "dist"},
+     {Assets.fonts(), "fonts"},
+     {Assets.markdown_processor_assets(), ""} | debug]
```

Comment:
I think we can skip this indirection and call `ExDoc.Markdown.get_markdown_processor().assets(:html)` directly?